app/templates/components/events/event-import-section.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="ui stackable centered grid">
+  <div class="sixteen wide column">
+    <h3 class="ui header">{{t 'Import Event'}}</h3>
```

Comment:
I don't think this title is needed. We already know this is the import page from the selected tab.